app/models/company.rb

```diff
@@ -2,5 +2,5 @@ class Company < ActiveRecord::Base
   validates :name, :organization, :title, :state, :city, presence: true
   validates :email, presence: true, length: { maximum: 255 },
                     format: { with: /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i, }
-
+  validates :hire_count, numericality: { only_integer: true, }
 end
```

Comment:
Nice!! I haven't seen numerically before! I was using regex, but this is very readable and easy to use!! :sparkles: